python-sdk/src/astro/databases/base.py

```diff
@@ -698,9 +700,7 @@ def export_table_to_file(
     # Schema Management
     # ---------------------------------------------------------
 
-    def create_schema_if_needed(
-        self, schema: str | None, location: str | None = None  # skipcq: PYL-W0613
-    ) -> None:
+    def create_schema_if_needed(self, schema: str | None) -> None:
```

Comment:
Is this a breaking change?

Reply:
This should have only been used internally (was added last week) and was not used by anything apart from Bigquery that too internally